docker_cli/setup_commands.py

```diff
@@ -270,65 +278,33 @@ def setup_conda_dev(
         The tag of the image to be built.
     no_cache : bool
         Run Docker build with no cache if True.
-    env_file : str
-        The location of the runtime environment spec-file.
+    env_file : Path
+        The location of the dev environment requirements file.
 
     Returns
     -------
     Image
         The generated image.
     """
+    # Set the context at the parent of the given environment file.
+    env_file_absolute = env_file.resolve()
+    context = env_file_absolute.parent
 
-    body = mamba_add_specs_dockerfile(env_specfile=env_file)
-
-    prefix = universal_tag_prefix()
-    img_tag = tag if tag.startswith(prefix) else f"{prefix}-{tag}"
-
-    full_dockerfile = f"FROM {base}\n\n{body}"
-
-    return Image.build(
-        tag=img_tag, dockerfile_string=full_dockerfile, no_cache=no_cache
-    )
-
-
-def setup_conda_add(
-    base: str,
-    tag: str,
-    no_cache: bool,
-    packages: Iterable[str],
-    channels: Optional[Iterable[str]],
-) -> Image:
-    """
-    Create an image that adds packages to another image's micromamba environment.
-
-    Parameters
-    ----------
-    base : str
-        The name of the image upon this one will be based.
-    tag : str
-        The tag of the image to be built.
-    no_cache : bool
-        Run Docker build with no cache if True.
-    packages : Iterable[str]
-        The packages to be added.
-    channels: Iterable[str], optional
-        Channels to look for packages in.
-
-    Returns
-    -------
-    Image
-        The generated image.
-    """
+    # Get the path to the environment file, relative to the context.
+    env_file_relative = env_file.resolve().relative_to(context.resolve())
```

Comment:
`env_file_absolute` and `context` are already absolute paths, so no need to do `resolve()` here at all.

```suggestion
    env_file_relative = env_file_absolute.relative_to(context)
```